torch/utils/hipify/hipify_python.py

```diff
@@ -1114,12 +1115,15 @@ def hipify(
         clean_ctx = GeneratedFileCleaner(keep_intermediates=True)
 
     # Preprocessing statistics.
-    stats: Dict[str, List] = {"unsupported_calls": [], "kernel_launches": []}
+    stats: Dict[str, List] = {"unsupported_calls": [], "kernel_launches": [], "skipped_no_permissions": []}
 
     for filepath in (all_files if not hipify_extra_files_only else extra_files):
         preprocess_file_and_save_result(output_directory, filepath, all_files, header_include_dirs,
                                         stats, hip_clang_launch, is_pytorch_extension, clean_ctx, show_progress)
 
+    num_skipped_no_permissions = len(stats["skipped_no_permissions"])
+    if num_skipped_no_permissions:
+        print(f"{bcolors.WARNING}Could not hipify {num_skipped_no_permissions} files due to no write permissions")
```

Comment:
If we don't care about the name of skipped files, and only care about the counter, we can use the int type, instead of a list[] . This will save space.  For example, use this instead: in line 1118: "num_skipped_no_permissions": int

Reply:
Actually, I was in two minds about whether to print the list or not. I feel like it would be useful to collate and print the list as a summary for the user. Thoughts?